challenge_5/python/tvanderwal/src/FindTheDifference.py

```diff
@@ -0,0 +1,19 @@
+class Solution:
+    def findTheDifference(self, s, t):
+        """
+        :param s: String
+        :param t: String
+        :return:  String
+        """
+
+        #Turn strings into sorted lists
+        sList = sorted(s)
+        tlist = sorted(t)
+
+        #enumerate over ever char in tList. If it isn't equal to the same spot in sList then it was added
+        for index,val in enumerate(tlist):
+            #If last value is the character aded
```

Comment:
I'll have to look into zip, still pretty green to python. Comprehension would be a really good idea here! Thanks for the heads up,